### PR TITLE
Fix measurement persistence.

### PR DESCRIPTION
**Pull request by benbjohnson**
Persist measurement metadata to the meta store.

### CODE REVIEW

File: shard.go [hunk 1]
```diff
@@ -120,6 +120,7 @@ func (s *Shard) writeSeries(seriesID uint32, timestamp int64, values []byte, ove
 		}
 
 		// Insert the values by timestamp.
+		warn("[write]", seriesID, time.Unix(0, timestamp))
```
Comment (otoolep) [inline shard.go]: You want this?

File: server.go [hunk 2]
```diff
@@ -230,6 +230,19 @@ func (s *Server) load() error {
 			}
 		}
 
+		// Open all shards.
+		for _, db := range s.databases {
+			for _, rp := range db.policies {
+				for _, g := range rp.shardGroups {
+					for _, sh := range g.Shards {
+						if err := sh.open(s.shardPath(sh.ID)); err != nil {
```
Comment (pauldix) [inline server.go]: we should only try to open the shard if the metadata for it says it exists on this server id

File: server.go [hunk 3]
```diff
@@ -230,6 +230,19 @@ func (s *Server) load() error {
 			}
 		}
 
+		// Open all shards.
```
Comment (otoolep) [inline server.go]: What is this change about?

File: shard.go (hunk 1 above)
Comment (benbjohnson) [inline shard.go]: For now, yeah. it's nice for debugging. :)

File: server.go (hunk 3 above)
Comment (benbjohnson) [inline server.go]: When the server starts up it just loads the shard metadata but doesn't actually open the underlying shard store. If you do a query before writing data to the shard after restart then it'll fail.

File: server.go (hunk 2 above)
Comment (benbjohnson) [inline server.go]: will fix later.

**Pull request merged**